README.adoc

```diff
@@ -2,13 +2,43 @@
 
 This repository contains Dockerfiles and scripts to build base images for use within Stackable.
 
+== Prerequisites
+
+* Stackable Image Tools (`pip install image-tools-stackabletech`) https://github.com/stackabletech/image-tools
+* Docker including the `buildx` plugin: https://github.com/docker/buildx
+* Optional: https://github.com/redhat-openshift-ecosystem/openshift-preflight[OpenShift preflight tool] to verify an image for OpenShift
+
+== Build Product Images
+
+Product images are published to the `docker.stackable.tech` registry under the `stackable` organization by default.
+
+To build and push product images to the default repository, use the `build_product_images.py` like this:
+
+    bake --product zookeeper --image 0.0.0-dev --push
+
+This will build images for Apache ZooKeeper versions as defined in the `conf.py` file, tag them with the `image-version` 0.0.0-dev and push them to the registry.
+
+You can select a specific version of a product to build using the syntax `product=version` e.g. to build Hive 3.1.3 you can use this command:
+
+    bake --product hive=3.1.3 -i 0.0.0-dev
+
+NOTE: `-i` is the shorthand for `--image`
```

Comment:
I'm not very familiar with `bake`, but is `-i` for just the image tag, or can you specify a whole new tag?

Eg: could I do `-i my.registry/image:dev`

```suggestion
NOTE: `-i` is the shorthand for `--image` (ie: the resulting image tag)
```

Reply:
Fixed in [`6e0cf20` (#581)](https://github.com/stackabletech/docker-images/pull/581/commits/6e0cf20eb078c394cc02a175ab5277c31b5fec8d)

Reply:
To answer your question: I have no idea myself. I'm just changing existing documentation here :(

Reply:
If only there were a --help option: